### PR TITLE
Unfucks lighting

### PR DESCRIPTION
**Pull request by ike709**
It's still a little fucked in some cases but it works at least 90% of the time now.

### CODE REVIEW

File: code/modules/lighting/lighting_object.dm [hunk 1]
```diff
@@ -139,7 +133,11 @@ GLOBAL_LIST_EMPTY(all_lighting_objects) // Global list of lighting objects.
 /atom/movable/lighting_object/blob_act()
 	return
 
+// Nope nope nope!
+/atom/movable/lighting_object/onShuttleMove(turf/T1, rotation)
```
Comment (ike709) [inline code/modules/lighting/lighting_object.dm]: I wonder if this affects us.

Comment (ninjanomnom) [inline code/modules/lighting/lighting_object.dm]: I'm guessing this just changes it to return FALSE instead of null but I'm not sure

**Comment by ninjanomnom** [comment]
@Vivalas take a look at this as well when you get the time

**Comment by Vivalas** [comment]
K

File: code/modules/lighting/lighting_corner.dm [hunk 2]
```diff
@@ -135,5 +137,5 @@ GLOBAL_LIST_INIT(LIGHTING_CORNER_DIAGONAL, list(NORTHEAST, SOUTHEAST, SOUTHWEST,
 		return QDEL_HINT_LETMELIVE
 
 	stack_trace("Ok, Look, TG, I need you to find whatever fucker decided to call qdel on a fucking lighting corner, then tell him very nicely and politely that he is 100% retarded and needs his head checked. Thanks. Send them my regards by the way.")
```
Comment (Vivalas) [inline code/modules/lighting/lighting_corner.dm]: memes

Comment (Vivalas) [inline code/modules/lighting/lighting_corner.dm]: Also why the fuck is Look capitalized I demand that be changed to lowercase "look" because grammar nazi.

**Pull request merged**